sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -36,6 +36,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\Azure.Messaging.EventHubs.Shared\src\Azure.Messaging.EventHubs.Shared.Core.projitems" Label="Core" />
   <Import Project="$(MSBuildThisFileDirectory)..\..\Azure.Messaging.EventHubs.Shared\src\Azure.Messaging.EventHubs.Shared.Diagnostics.projitems" Label="Diagnostics" />
   <Import Project="$(MSBuildThisFileDirectory)..\..\Azure.Messaging.EventHubs.Shared\src\Azure.Messaging.EventHubs.Shared.Authorization.projitems" Label="Authorization" />
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Azure.Messaging.EventHubs.Shared\src\Azure.Messaging.EventHubs.Shared.Processor.projitems" Label="Processor" />
```

Comment:
I'm quite likely overlooking something, but what does the core Event Hubs library need from the shared processor items?   Those should be `internal` types and the only processor things in this library should be the public event argument types.

Reply:
It was due to the relocation of the `PartitionManager` class. Without this reference, we fail to build. If you see an alternative to this, please let me know.

Reply:
That confuses me.   There's no reference that I can see (in master, at least) of the `PartitionManager` in the core library.   Since there's a "Processor depends on core" relationship and partition manager lives in the processor library, there's no way that it could be reference in the core library without a circular dependency.... right?    

I feel like I'm overlooking something important.

Reply:
Ok.  I think that I get it.   It's the inclusion of `MockCheckPointStorage` in the `testing` bundle that pulls that in.

Would you be opposed to creating a "Processor.Testing` bundle to keep that line for references crisp?

Reply:
sure thing! I defer to your expertise on the project structure.

Reply:
Much appreciated!

Reply:
I realized that the `Processor` bundle reference was only needed in the Azure.Messaging.EventHubs.Tests.csproj so I just moved it there rather than creating another bundle for the same scope. Let me know if that isn't ok.

Reply:
Thanks!  That should be fine.   In fact, as I was looking through some future plans for extensibility this morning, I realized that I'm overlooking that we're likely to need the processor references in the core package in the near future.  Apologies for the short-sighted perspective there.